node/runtime/pangolin/src/pallets/bridge_rococo_grandpa.rs

```diff
@@ -0,0 +1,28 @@
+pub use pallet_bridge_grandpa::Instance2 as WithRococoGrandpa;
+
+// --- paritytech ---
+use drml_bridge_primitives::Rococo;
+use pallet_bridge_grandpa::Config;
+// --- darwinia-network ---
+use crate::*;
+use pangoro_primitives::DAYS;
+
+frame_support::parameter_types! {
+	// This is a pretty unscientific cap.
+	//
+	// Note that once this is hit the pallet will essentially throttle incoming requests down to one
+	// call per block.
+	pub const MaxRequests: u32 = 50;
+	// Number of headers to keep.
+	//
+	// Assuming the worst case of every header being finalized, we will keep headers for at least a
+	// week.
+	pub const HeadersToKeep: u32 = 7 * DAYS as u32;
+}
+
+impl Config<WithRococoGrandpa> for Runtime {
+	type BridgedChain = Rococo;
+	type MaxRequests = MaxRequests;
+	type HeadersToKeep = HeadersToKeep;
+	type WeightInfo = ();
```

Comment:
@AurevoirXavier How to config the WeightInfo? Are we OK to set `()`?

Reply:
`()` means default.

Before we bench this pallet, I think we can use the default value.